Sources/MapboxCoreNavigation/TilesetDescriptorFactory.swift

```diff
@@ -9,13 +9,16 @@ extension TilesetDescriptorFactory {
      - Parameters:
        - version: TilesetDescriptor version.
        - completionQueue: A DispatchQueue on which the completion will be called.
+       - datasetProfile: Profile setting, used for selecting tiles type for navigation.
        - completion: A completion that will be used to pass the tileset descriptor.
      */
     public class func getSpecificVersion(version: String,
```

Comment:
Does the fact that TilesetDescriptorFactory is defined in MapboxNavigationNative prevent these methods from showing up in the docset?

Reply:
What do you mean? I see there are [references in the doc](https://docs.mapbox.com/ios/navigation/api/2.2.0/Extensions/TilesetDescriptorFactory.html#/s:So28MBNNTilesetDescriptorFactoryC20MapboxCoreNavigationE18getSpecificVersion7version15completionQueue0K0ySS_So17OS_dispatch_queueCySo010MBXTilesetB0CctFZ).